### PR TITLE
Support build on jdk 11

### LINKED ISSUE #70: Build fails on jdk 11
1.5 is specified as target release for compiler, while jdk 11 supports only 6 and above. 
```
[INFO] -------------------------------------------------------------
[ERROR] COMPILATION ERROR : 
[INFO] -------------------------------------------------------------
[ERROR] error: Source option 5 is no longer supported. Use 6 or later.
[ERROR] error: Target option 1.5 is no longer supported. Use 1.6 or later.
```

### PR DESCRIPTION
**Pull request by andreoss**
Closes #70

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -135,7 +135,7 @@
       <plugin>
         <groupId>net.alchim31.maven</groupId>
         <artifactId>scala-maven-plugin</artifactId>
-        <version>${scala.maven.plugin.version}</version>
+        <version>4.3.0</version>
```
Comment (katrinsharp) [inline pom.xml:138]: What was the reason to get rid of `scala.maven.plugin.version` variable?

File: pom.xml [hunk 2]
```diff
@@ -270,7 +270,7 @@
           <docencoding>UTF-8</docencoding>
           <docfilessubdirs>true</docfilessubdirs>
           <show>package</show>
-          <source>1.5</source>
+          <source>1.${java.version}</source>
```
Comment (katrinsharp) [inline pom.xml:273]: Perhaps to make the whole version as a variable?

Comment (andreoss) [reply pom.xml:273]: Since it's outdated versioning scheme for JDK, I preferred leaving `1.` from main variable

File: pom.xml (hunk 1 above)
Comment (andreoss) [reply pom.xml:138]: It appears that other plugins around do not have a dedicated version variable, so I've inlined it.
Is it set from outside by something (by some integration test perhaps)?

**Comment by katrinsharp** [comment]
LGTM

**Pull request merged**